inductiva/__init__.py

```diff
@@ -66,3 +66,33 @@ def _check_for_available_package_update():
 
 
 _check_for_available_package_update()
+
+
+def _supports_ansi():
+    """Checks if we support ansi formatting for colors and bolds"""
+    user_disable_ansi = utils.format_utils.getenv_bool("INDUCTIVA_DISABLE_ANSI",
```

Comment:
Time to open a issue and start to listing these.

Reply:
https://github.com/inductiva/tasks/issues/92